### DISCUSSION TITLE
How to reuse data after recreate milvus by ansible?

### DISCUSSION THREAD

**Discussion by Derekhe01** [milvus-io/milvus#17487] [Q&A and General discussion]
I have map the volumn of minio and etcd. 
minio:
    volumes:
      - /opt/minio_data:/minio_data
etcd:
    volumes:
     - /opt/etcd:/etcd 

But when I recreate by ansible, proxy have an error:
[WARN] [grpclog.go:46] ["[core]grpc: Server.Serve failed to create ServerTransport: connection error: desc = \"transport: http2Server.HandleStreams failed to receive the preface from client: EOF\""]

**Comment by yhmo** [comment]
How did you deploy the milvus?
